tycho/template/pod.yaml

```diff
@@ -68,6 +68,21 @@ spec:
     volumeMounts:
     - name: {{ system.stdnfs_pvc }}
       mountPath: {{ system.parent_dir }}
+  {% if system.irods_enabled == True %}
+  - name: nfsrods-prep
+    image: busybox:1.28
+    # Fixed resources
+    command: ['sh', '-c'] 
+    args:
+    - touch /permissions/irodsvalidusers &&
```

Comment:
Why is this needed?

Reply:
This is how I chose to solve the problem that the permissions file needs to be written at a higher privilege level than the user has access to so they can't change the UID. The initContainer runs this script as root.

Reply:
Right after this you echo a line to the same file, which would create it if it didn't exist.  The permissions wouldn't change either way.  Am I missing something?

Reply:
Ah! I see what you mean. I'll remove that.